<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1631</width>
    <height>1039</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="videolabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>1600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgray;
text-align:center</string>
    </property>
    <property name="text">
     <string>NO VIDEO LOADED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>503</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Playback</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QPushButton" name="fastfwdButton">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string> ||</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="rewindButton">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QSlider" name="videoSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>820</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Motion Estimator</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>53</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="est_x">
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="est_y">
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>820</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>VideoReader</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Frame:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="curr_frame">
         <property name="text">
          <string>0/0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>820</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Crack Inspector</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="jumpToNextCrackButton">
       <property name="text">
        <string>Jump to next crack</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>820</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QCheckBox" name="cbPatches">
       <property name="text">
        <string>Patches</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="cbNaiveBayes">
       <property name="text">
        <string>Naive Bayes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="cbMotionEstimator">
       <property name="text">
        <string>M.E.</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>830</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QSpinBox" name="spinNBThreeshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>N.B. Threeshold</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_6</zorder>
   <zorder>videolabel</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1631</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_video_footage"/>
    <addaction name="actionSelect_SPR_file"/>
    <addaction name="actionExit_Application"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionSelect_Output_Video"/>
    <addaction name="actionEnable_capture_to_file"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCapture"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_video_footage">
   <property name="text">
    <string>Select video footage</string>
   </property>
  </action>
  <action name="actionSelect_SPR_file">
   <property name="text">
    <string>Select SPR file</string>
   </property>
  </action>
  <action name="actionExit_Application">
   <property name="text">
    <string>Exit Application</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionSelect_Output_Video">
   <property name="text">
    <string>Select Output Video</string>
   </property>
  </action>
  <action name="actionEnable_capture_to_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable capture to file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
